docs/content/docs/getting-started.md

```
@@ -1,21 +1,53 @@
 You need [`sqlc`](https://docs.sqlc.dev/en/latest/overview/install.html) on your
-`PATH`. The generated code targets **Python 3.12 or newer** (it uses PEP 695 type
-aliases and generics, and `enum.StrEnum`).
+`PATH` and **Python 3.12 or newer** (the generated code uses PEP 695 type aliases
+and generics, and `enum.StrEnum`).
+
+Then install the database driver you want to use:
+
+{{< tabs >}}
+
+  {{< tab name="asyncpg" >}}
+```bash
```

markdownlint-cli2 (0.23.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)